gno.land/pkg/gnoclient/client_txs.go

```diff
@@ -145,11 +144,6 @@ func (c *Client) Run(cfg BaseTxCfg, msgs ...MsgRun) (*ctypes.ResultBroadcastTxCo
 
 		caller := c.Signer.Info().GetAddress()
 
-		// Transpile and validate Gno syntax
-		if err = transpiler.TranspileAndCheckMempkg(msg.Package); err != nil {
```

Comment:
I am actually deeply disturbed we had this kind of check in a client method

Reply:
For reference, we don't (actually, can't) do this kind of validation on our JS clients

Reply:
What do you mean, we don't ship a wasm-compiled blob of the `gofmt` binary to client-side check the syntax of the gno program?!